_episodes/02-filedir.md

```diff
@@ -401,8 +401,8 @@ $ man ls
 {: .language-bash}
 
 This will turn your terminal into a page with a description
-of the `ls` command and its options and, if you're lucky, some examples
-of how to use it.
+of the `ls` command and its options and, depending on your version of
+`ls` and its documentation, some examples of how to use it.
```

Comment:
Thanks for the suggestion here.

I think I'd like something like "depending on your variant of the terminal software supplying `ls` and its documentation'

Reply:
Would a mix of terminal software and operating system be more appropriate?

"depending on your variant of the terminal software, or your operating system, supplying `ls` and its documentation”

I’d consider Git Bash as terminal software and shell. Using macOS for the operating system, using either Terminal or iTerm2, and either bash or zsh, the same ls binary and man page content are used for `ls`.

Reply:
Sounds fine with me.

Reply:
I would be in favour of a very short explanation here (or even no explanation because it's not a key point). All we want to convey to learners is that the man page they see for `ls` won't necessarily have any examples. Could we not just change 'if you're lucky' to 'maybe'? Or maybe just cut the current sentence short
> This will turn your terminal into a page with a description of the ls command and its options.